recipes/smif/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "smif" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "a806ef66b785e1620300bc9ce55a5488fc77d393ea3175db661cc741e234acf0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - smif = smif.cli:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyscaffold
+  run:
+    - fiona 1.7.10
+    - flask 0.12.2
+    - isodate 0.6.0
+    - networkx 1.11
+    - numpy 1.13.1
+    - pint 0.8.1
+    - python
+    - python-dateutil 2.6.1
+    - pyyaml 3.12
+    - rtree 0.8.3
+    - scikit-optimize 0.3
+    - shapely 1.6.2
+
+about:
+  home: http://github.com/nismod/smif
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Simulation Modelling Integration Framework'
+  description: |
+    smif is a framework for handling the creation of system-of-systems models.
+    The framework handles inputs and outputs, dependencies between models,
+    persistence of data and the communication of state across years. This early
+    version of the framework handles simulation models that simulate the
+    operation of a system within a year. smif exposes an interface to a planning
+    module which will allows different algorithms to be used against a common
+    API.
+  doc_url: http://smif.readthedocs.io/
+  dev_url: https://github.com/nismod/smif
```

Comment:
You need to add at least yourself as a maintainer:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L70-L75

Reply:
Done. Thank you for the initial review!